src/components/CircleIcon/types.ts

```diff
@@ -0,0 +1,5 @@
+export interface CircleIconProps {
+  src: string;
+  customClass?: string;
+  isHighlight?: boolean;
```

Comment:
I think isHighlight should be mandatory as we want to tell our component to highlight our icon (if user has active story) or not.

Reply:
But there is no point to use isHighlight prop as false cuz if we don't get this prop it's false by default, so it would be redundance

Reply:
Yes, but both customClass and isHighlight will be undefined and it won't match our interface, so it would be cool to set default values for them. For example this line of code:
```<div className={`relative w-6 mx-2 cursor-pointer ${customClass}`}>```
will be compiled to 
```<div class='relative w-6 mx-2 cursor-pointer undefined'>```